test/AppNotificationTests/AppNotificationHelpers.h

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+namespace winrt
+{
+    using namespace winrt::Windows::ApplicationModel::Activation;
+    using namespace winrt::Windows::Foundation;
+    using namespace winrt::Microsoft::Windows::AppNotifications;
+    using namespace winrt::Windows::Foundation;
+}
+
+namespace AppNotificationHelpers {
+    winrt::AppNotification CreateToastNotification(winrt::hstring message)
+    {
+        winrt::hstring xmlPayload{ L"<toast>" + message + L"</toast>" };
+        return winrt::AppNotification(xmlPayload);
+    }
+
+    winrt::AppNotification CreateToastNotification()
+    {
+        return CreateToastNotification(L"intrepidToast");
+    }
+
+    void VerifyToastStatus(UINT32 expectedToastId, bool expected)
+    {
+        auto retrieveNotificationsAsync{ winrt::AppNotificationManager::Default().GetAllAsync() };
+        auto scope_exit = wil::scope_exit(
+        [&] {
+            retrieveNotificationsAsync.Cancel();
+        });
+
+        VERIFY_ARE_EQUAL(retrieveNotificationsAsync.wait_for(std::chrono::seconds(300)), winrt::AsyncStatus::Completed);
+        scope_exit.release();
+
+        auto notifications{ retrieveNotificationsAsync.get() };
+
+        bool found{ false };
```

Comment:
NIT: `bool found{};`

No need for explicit value when it's redundant with the default (primitives don't need `{ false/0/0.0/nullptr/etc }` just use `{}`).